chia/rpc/data_layer_rpc_api.py

```diff
@@ -159,8 +159,8 @@ async def get_root(self, request: Dict[str, Any]) -> Dict[str, Any]:
         # todo input checks
         if self.service is None:
             raise Exception("Data layer not created")
-        res = await self.service.get_root(store_id)
-        return {"hash": res}
+        hash, status = await self.service.get_root(store_id)
+        return {"hash": hash, "status": status}
```

Comment:
I had this thought yesterday, but maybe it's worth asking in light of `too many values to unpack` posted by @emlowe: Is there a convention in the code base w.r.t. getters called with `await` when they return pairs? Since when the result is is always a "maybe type", those returning `None` makes parsing pairs a bit trickier. 
(Given with how `get_root` is used I though it could make that function `return dict(hash=hash, status=status)` directly, and potentially fuse dicts with `update`. But also here I don't know all the conventions used in the codebase.)

Reply:
We use tuples regularly.  I would personally rather see custom class instances to avoid the order dependence of tuples and the vagueness of dicts.  And yes, there is `TypedDict` but by the time you define one you've literally written a class and you could have used `@dataclass` instead and had a real class.  Lots of options are discussed at https://www.attrs.org/en/stable/why.html.  (dataclasses is a limited fork of attrs into the stdlib)

But, I'm trying to leave that for a dedicated discussion.  And sure, I often fail to keep quiet about it.  `:|`